dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js

```diff
@@ -188,5 +223,293 @@ describe('Sign In Gate Tests', function () {
 
 			cy.contains('privacy settings');
 		});
+
+		describe('Sign in gate should personalise based on the GU_CO_COMPLETE cookie', function () {
+			it('should show the main sign in gate if GU_CO_COMPLETE if not present', function () {
+				postArticleAndScrollToGateForLazyLoad({
+					switchOverride: {
+						personaliseSignInAfterCheckoutGate: true,
+					},
+				});
+
+				cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
+				cy.get('[data-cy=sign-in-gate-main]').contains(
+					'You need to register to keep reading',
+				);
+				cy.get('[data-cy=sign-in-gate-main]').contains(
+					'It’s still free to read – this is not a paywall',
+				);
+				cy.get('[data-cy=sign-in-gate-main]').contains(
+					'We’re committed to keeping our quality reporting open.',
+				);
+				cy.get('[data-cy=sign-in-gate-main_register]').contains(
+					'Register for free',
+				);
+			});
+
+			it('should show the main sign in gate if GU_CO_COMPLETE is present but flag is false', function () {
+				setGuCOCompleteCookie('new', 'DigitalPack');
+
+				postArticleAndScrollToGateForLazyLoad({
+					switchOverride: {
+						personaliseSignInAfterCheckoutGate: false,
+					},
+				});
+
+				cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
+				cy.get('[data-cy=sign-in-gate-main]').contains(
+					'You need to register to keep reading',
+				);
+			});
+
+			it('should show the main sign in gate if GU_CO_COMPLETE is present but flag is not set', function () {
+				setGuCOCompleteCookie('new', 'DigitalPack');
+
+				postArticleAndScrollToGateForLazyLoad();
+
+				cy.get('[data-cy=sign-in-gate-main]').should('be.visible');
+				cy.get('[data-cy=sign-in-gate-main]').contains(
+					'You need to register to keep reading',
+				);
+			});
+
+			describe('Sign in gate should show personalised copy if GU_CO_COMPLETE is present', function () {
+				// HEADER TEXT
```

Comment:
Yeah I did think about bringing these in from source. I didn't initially because the cypress project doesn't recognise files from source (though I did end up figuring that out for the switching logic). But also think it's maybe better to keep them separate as the test is defining what the answer should be, so feels more explicit to write here.